### DISCUSSION TITLE
Gas disk and/or spherical environment

### DISCUSSION THREAD

**Discussion by kufapetr** [dullemond/radmc3d-2.0#32] [General]
Hello all, I am new in RADMC-3D and have the following question: I am trying to calculate the light curve of the star embedded in a more or less gaseous spherical circumstellar environment from different selected viewing angles. I have calculated the 3D density, velocity, and temperature structures from a different code and used them as inputs. That is clear so far. Anyway, if I use the "gas" examples, there is always necessary to implement the dust environment. Is it possible to eliminate dust or at least suppress it as much as possible and calculate the line-of-sight directed optical depth for gas only (with selected chemical composition)? I notice that the "gas examples" do not allow calculating SED but only line spectra. 

Thank you, 
Petr

**Comment by dullemond** [comment]
I think it is important that you specify the physics of the gas. What are the temperature and density? Which wavelength range are you interested in? RADMC-3D is designed for tenuous and relatively cold media, such as circumstellar disks, protostellar molecular clouds, cold ISM etc. Individual lines can be well-modelled in LTE, such as CO rotational lines. But RADMC-3D is not designed to do millions of rovibrational lines in the near-infrared, so it would not work (at least, not out-of-the-box) for exoplanet atmospheres or complex environments such as PDRs or so. These are possible, but they would require quite a bit of do-it-yourself work.
